template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPageWithPageModel/Index.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
```

Comment:
Is this a good default set of namespaces? Anything else common that should go here? (Not at all saying it isn't, I'm just not familiar with what's generally used.)